code/lib/preview-api/src/modules/core-client/start.test.ts

```diff
@@ -106,6 +112,12 @@ function makeRequireContext(importMap: Record<Path, ModuleExports>) {
 describe('start', () => {
   beforeEach(() => {
     global.DOCS_OPTIONS = { enabled: false };
+    // @ts-expect-error (setting this to undefined is indeed what we want to do)
```

Comment:
Is this an indication that `global.__STORYBOOK_CLIENT_API__` is typed incorrectly?  I think nearly all globals should be allowed to be `undefined`, since they will be undefined until they are set somewhere.

Reply:
TypeScript doesn't like it:

![Screenshot 2022-12-13 at 19 06 16](https://user-images.githubusercontent.com/3070389/207411049-56999cdc-7231-4d7c-bc3a-c566b7f24472.png)

reason being:
https://github.com/storybookjs/storybook/blob/adb0d25092c0794333fb82864d033e8a26cda668/code/lib/preview-api/src/typings.d.ts#L27-L28

I could set this to ` | undefined`, sure... Do you think that's better?

Reply:
>I could set this to | undefined, sure... Do you think that's better?

I _think_ so.  That's what I did in my PR, to indicate that these values aren't always there, they have to be set during runtime, which means there's a period of time in which they are undefined.  But maybe that's being overly pedantic.